openlibrary/templates/account/mybooks.html

```diff
@@ -14,25 +17,24 @@
   </span>
 </div>
 
-$if user.is_beta_tester():
-  $ current_goal = get_reading_goals()
-  $ hidden = 'hidden' if current_goal else ''
+$ year = 2023
```

Comment:
We may want something slightly smarter than this (or next year we could forget)

e.g. some function which returns year if month not 12 else year+1